test/test_datasets.py

```diff
@@ -512,7 +512,7 @@ def inject_fake_data(self, tmpdir, config):
         return dict(num_examples=num_images_per_split[config["split"]], attr_names=attr_names)
 
     def _create_split_txt(self, root):
-        num_images_per_split = dict(train=3, valid=2, test=1)
+        num_images_per_split = dict(train=3, valid=2, test=2)
```

Comment:
Let's us a different value for each split to make sure everything works as expected.

```suggestion
        num_images_per_split = dict(train=4, valid=3, test=2)
```